react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,33 +1,36 @@
 import React, { Component } from 'react';
-
 import Square from '../Square';
-
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
+
+  squarePosition(i) {
+    return (
+      <Square
+        value={this.props.squares[i]}
+        onClick={() => this.props.onClick(i)}
```

Comment:
In onClick you can send this.props.onClick and send 'i' like a prop. With this, you don't need to create an arrow function inside where you send a prop.